style.css

```diff
@@ -1,250 +1,530 @@
 body {
   font-family: 'montserratsemibold';
   font-style: normal;
+  overflow: auto;
 }
 
 header {
-display: flex;
-align-items: center;
-width: 1800px;
-height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 40px;
+  padding-right:10px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)